### PR TITLE
update docs to use new platform attributes in supports

### PR DESCRIPTION
**Pull request by chris-rock**
Signed-off-by: Christoph Hartmann <chris@lollyrock.com>

### CODE REVIEW

File: docs/profiles.md [hunk 1]
```diff
@@ -119,13 +121,17 @@ and to target all of these examples in a single `inspec.yml` file:
...
-  - os-name: ubuntu
+  - platform-name: debian
+  - platform-name: ubuntu
     release: 14.04
-  - os-family: redhat
+  - platform-family: redhat
   - platform: aws
 ```
 
+With InSpec 2.0, we introduced the `platform` attribute, which allows to restrict the execution to `os`, `aws`, `azure` or `gcp`
```
Comment (clintoncwolfe) [inline docs/profiles.md]: Do we actually have an attribute literally named `platform`?  This might be confusing

File: docs/profiles.md [hunk 2]
```diff
@@ -81,20 +81,22 @@ $ inspec check examples/profile
 
 # Platform Support
 
-Use the `supports` setting in the `inspec.yml` file to specify one (or more) platforms for which a profile is targeting. The list of supported platforms may contain simple names, names and versions, or detailed flags, and may be combined arbitrarily. For example, to target anything running Debian Linux:
+Use the `supports` setting in the `inspec.yml` file to specify one (or more) platforms for which a profile is targeting. The list of supported platforms may contain simple names, names and versions, or detailed flags, and may be combined arbitrarily.
```
Comment (clintoncwolfe) [inline docs/profiles.md]: Could we have an explicit list of the keys that are allowed here? ` may contain simple names, names and versions, or detailed flags` does not tell what is allowed and what isn't.  Examples are good, but don't tell you the complete picture.

**Review by clintoncwolfe — DISMISSED** [review]
Thanks, @chris-rock ! Just some clarifications

File: docs/profiles.md (hunk 1 above)
Comment (jquick) [reply docs/profiles.md]: Yes we do have `platform` which under the hood is if platform-name OR platform-family are true. I'll re-write this to be a bit more clear.

**Comment by chris-rock** [comment]
Thank you @clintoncwolfe for the questions and @jquick for the clarification

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Pull request merged**